src/document_handler.rb

```diff
@@ -0,0 +1,32 @@
+require 'telegramAPI'
+
+class DocumentHandler
+  attr_reader :document, :api, :url
+
+  def initialize(request_handler)
+    @token = ENV['TOKEN']
+    @document = request_handler.message.document
+
+    @api = TelegramAPI.new(@token)
+    @url = "https://api.telegram.org/file/bot#{@token}/#{api.getFile(document.file_id)['file_path']}"
+  end
+
+  def translate_document
+    File.new('translated.txt', 'w').puts translate_content
```

Comment:
Um, it won't do anything.
Plus, you have to close files after you open them.
In Ruby you can use blocks to let Ruby close them for you.
`File.open('translated.txt', 'w') { |file| file.write(translate_content.text) }`